src/api/index.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+export const http = axios.create({
+  baseURL: 'https://zinnia-ah-backend-staging.herokuapp.com/api/v1',
+});
+
+export const setToken = token => {
+  if (getToken()) {
```

Comment:
The function, `localStorage.setItem` would replace the previous value of token. There is no need to call `destroyToken` since all it does is to run `localStorage.removeItem()`